packages/base-driver/index.js

```diff
@@ -60,3 +60,8 @@ export { processCapabilities, isStandardCap, validateCaps };
 import * as ws from './lib/express/websocket';
 const { DEFAULT_WS_PATHNAME_PREFIX } = ws;
 export { DEFAULT_WS_PATHNAME_PREFIX };
+
+// Arguments
+import * as args from './lib/basedriver/arguments';
```

Comment:
we could import parseServerArgs directly: `import { parseServerArgs } from './lib/basedriver/arguments';`